src/AppBundle/Controller/BlogController.php

```diff
@@ -36,12 +36,18 @@ class BlogController extends Controller
     /**
      * @Route("/", name="blog_index")
      */
-    public function indexAction()
+    public function indexAction(Request $request)
     {
-        $em = $this->getDoctrine()->getManager();
-        $posts = $em->getRepository('AppBundle:Post')->findLatest();
```

Comment:
@javiereguiluz the issue with this is that KnpPaginator is a service doing the pagination. so using it in the Doctrine repository makes things complex (this could be an argument in favor of Pagerfanta btw)